packages/itmat-interface/src/lxd/index.ts

```diff
@@ -0,0 +1,325 @@
+import express from 'express';
+import { WebSocketServer, WebSocket } from 'ws';
+import qs from 'qs';
+import lxdManager from './lxdManager';
+
+import jwt from 'jsonwebtoken';
+import { GraphQLError } from 'graphql';
+import { ApolloServerErrorCode } from '@apollo/server/errors';
+import { userRetrieval, userRetrievalByUserId } from '../authentication/pubkeyAuthentication';
+import { logRESTAPICall } from '../log/restLogHelper';
+
+const lxdCallsRouter = express.Router();
+
+lxdCallsRouter.use(logRESTAPICall);

```

Comment:
Shouldn't that already be done upper in the router stack ? There is already a `LogPlugin` for GraphQL, probably best to uniformalise them